Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -92,6 +91,21 @@ function validateProps(params: Object): void {
   }
 }
 
+function validateTransform(config: Object): void {
+  var TRANSFORM_WHITELIST = {
+    translateX: true,
+    translateY: true,
+    scale: true,
+    rotate: true
+  };
+
+  for (var key in config) {
+    if (!TRANSFORM_WHITELIST.hasOwnProperty(key)) {
```

Comment:
we might want to move the white list outside of the actual function call. Then you could maybe also just do `TRANSFORM_WHITELIST !== true` :shrug: